modules/sc-mesh-secure-deployment/src/nats/run_unittests_PC.sh

```diff
@@ -28,6 +28,8 @@ source unittest/bin/activate
 # install dependencies to virtualenv
 pip install coverage==7.4.4  # this is for testing purpose
 pip install -r requirements.txt
+pip install pyroute2==0.7.12  # This version is needed on Ubuntu
+pip install parameterized==0.9.0  # Used in unit tests only
```

Comment:
If `parameterized` is only used in unit tests I'd recommend creating a `requirements.txt` under the `tests` folder with it (and maybe `coverage`) :coat: